src/redux/epics/index.js

```diff
@@ -107,15 +107,21 @@ export const selectResourceType = (resourceType) => ({
   payload: resourceType,
 });
 
-export const addResourceToCollection = (collectionId, resourceIds) => ({
-  type: actionTypes.ADD_RESOURCE_TO_COLLECTION,
-  payload: { collectionId, resourceIds },
-});
+export const addResourceToCollection = (collectionId, resourceIds) => {
+  const payloadIds = Array.isArray(resourceIds) ? resourceIds : [resourceIds];
+  return ({
+    type: actionTypes.ADD_RESOURCE_TO_COLLECTION,
+    payload: { collectionId, resourceIds: payloadIds },
+  });
+};
 
-export const removeResourceFromCollection = (collectionId, resourceIds) => ({
-  type: actionTypes.REMOVE_RESOURCE_FROM_COLLECTION,
-  payload: { collectionId, resourceIds },
-});
+export const removeResourceFromCollection = (collectionId, resourceIds) => {
+  const payloadIds = Array.isArray(resourceIds) ? resourceIds : [resourceIds];
```

Comment:
should the caller just send an Array?

...might be in `ResourceCard` :
`addResourceToCollection(selectedCollectionId, resourceId)}`

(Should this button be a dedicated component?)

Reply:
`addResourceToCollection(selectedCollectionId, resourceId)}` this is def what is causing the break. I figure since this button would always add a single id, using `[resourceId]` might be confusing.

Yes, this could probably be a dedicated component.